README.md

```diff
@@ -28,15 +28,28 @@ like the Clang-as-a-library compiler implementation.
 
 libVersioningCompiler uses the cmake build system.
 If you want to install libVersioningCompiler
-
+install the dependencies with LLVM version `${LLVM_V}`:
+
+```bash
+wget https://apt.llvm.org/llvm.sh
+chmod +x llvm.sh
+[sudo] ./llvm.sh ${LLVM_V} all
+[sudo] apt-get install -y git cmake uuid-dev g++-10 libz-dev
+[optional][sudo] update-alternatives --install /usr/bin/clang++ clang++ /usr/bin/clang++-15 100
+[optional][sudo] update-alternatives --install /usr/bin/clang clang /usr/bin/clang-15 100
+[optional][sudo] update-alternatives --install /usr/bin/opt opt /usr/bin/opt-15 100
```

Comment:
IIRC these can be manually set in the source code.

In the long run, the configure system (cmake) can automatically detect them and set some default values.
As of now, however, these alias are not really required neither beneficial. Unless you also changed something that relies on them elsewhere.

Reply:
Without those on Ubuntu 20.04 LTS I had some internal errors (in libc-main and in lto/lto-partition.c) in the linking process which were practically undebuggable. I suspected they happened due to incoherent versions of clang with opt but I couldn't find the culprit. IMHO I have poorly set up Ubuntu, resulting in a broken configuration, that's why I had added those commands.

Reply:
If you can reproduce the error in a clean environment, we can have a call and try to solve the issue together. This solution may work, but is sounds not the appropriate way to handle it.